src/fastoad/models/performances/mission/polar.py

```diff
@@ -11,15 +11,31 @@
 #  GNU General Public License for more details.
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
-from numpy import ndarray
+import numpy as np
 from scipy.interpolate import interp1d
 from scipy.optimize import fmin
 
 
 class Polar:
-    def __init__(self, cl: ndarray, cd: ndarray):
 
+    _use_ground_effect = False
+    _use_CL_alpha = False
+
+    gnd_effect_variables_raymer = {
+        "span": {"name": "data:geometry:wing:span", "unit": "m"},
+        "lg_height": {"name": "data:geometry:landing_gear:height", "unit": "m"},
+        "induced_drag_coef": {
+            "name": "data:aerodynamics:aircraft:low_speed:induced_drag_coefficient",
+            "unit": None,
+        },
+        "k_winglet": {
+            "name": "tuning:aerodynamics:aircraft:cruise:CD:winglet_effect:k",
+            "unit": None,
+        },
+        "k_cd": {"name": "tuning:aerodynamics:aircraft:cruise:CD:k", "unit": None},
+    }
+
+    def __init__(self, input_arg=None):
```

Comment:
Fixed in 36bab9f464f2e077bc2e86f938d1c440b7c47158 , now the argument can only be to be a dictionary but it ought to be a generic input name as it can contain different variables depending on the ground effect and takeoff options